javascript/ql/lib/semmle/javascript/Arrays.qll

```diff
@@ -446,12 +446,12 @@ private module ArrayLibraries {
   }
 
   /**
-   * A taint propagating data flow edge arising from sorting.
+   * A taint propagating data flow edge arising from in-place array manipulation operations.
```

Comment:
Maybe here add that the methods also return the pointer to `this`.

Reply:
Resolved in 38d3e374fb6fb8d21c986d8f6df257770ccf9dd3.